src/img.c

```diff
@@ -1,6 +1,5 @@
 #include <stdio.h>
 #include <stdlib.h>
-#include <png.h>
```

Comment:
You may want to leave this. It makes it explicitly clear that png.h has been
included.